candig/server/sqlite_backend.py

```diff
@@ -18,7 +15,7 @@ def sqliteRowsToDicts(sqliteRows):
     :param sqliteRows: array of rows returned from fetchall DB call
     :return:  array of dicts, keyed by the column names.
     """
-    return map(lambda r: dict(zip(r.keys(), r)), sqliteRows)
+    return [dict(list(zip(list(r.keys()), r))) for r in sqliteRows]
```

Comment:
This looks very complicated to me. Can we use the sqliteRowToDict method instead and just iterate through the rows?

Reply:
This is probably a bug with the auto-converter

Reply:
Inner list is removed